Authentication/LoginProviders/Facebook/FacebookLoginProvider.swift

```diff
@@ -64,6 +64,12 @@ public struct FacebookLoginProviderUser: LoginProviderUser {
         self.userId = userId
         self.accessToken = accessToken
     }
+
+    public func logOut() {
+        let loginManager = LoginManager()
```

Comment:
Yes, it does a facebook logout, the button will change to Continue with Facebook (which is their "log in")